examples/TestClient.java

```diff
@@ -0,0 +1,526 @@
+	
+	private GengoClient gc;
+	private String sessionHash;
+	
+	/**
+	 * 
+	 * @param baseUrl Set one of GengoConstants.BASE_URL_*.
+	 */
+	public TestClient(ApiServiceLevel level) {
+		switch(level) {
+			case STANDARD:
+				this.gc = new GengoClient(API_KEY_PUBLIC, API_KEY_PRIVATE);
+				break;
+			case SANDBOX:
+				this.gc = new GengoClient(API_KEY_PUBLIC, API_KEY_PRIVATE, true);
+				break;
+			default:
+				throw new IllegalArgumentException("level is missing.");
+		}
+		this.sessionHash = new Random().toString();
+	}
+	
+	/**
+	 * Test account functionalities.
+	 */
+	public void testAccount(Boolean verbose) {
+		verbose = verbose == null ? true : verbose;
+		System.out.println("[testAccount] start@" + this.sessionHash);
+		JSONObject response = null;
+		// Stats(GET)
+		System.out.println("Account::Stats(GET)");
+		try {
+			response = this.gc.getAccountStats();
+			System.out.print("[OK] ");
+			System.out.println(verbose ? response.toString() : "");
+		} catch (GengoException x) {
+			System.out.print("[NG] ");
+			System.out.println(x.getMessage());
+		}
+		// Balance(GET)
+		System.out.println("Account::Balance(GET)");
+		try	{
+			response = this.gc.getAccountBalance();
+			System.out.print("[OK] ");
+			System.out.println(verbose ? response.toString() : "");
+		} catch (GengoException x) {
+			System.out.print("[NG] ");
+			System.out.println(x.getMessage());
+		}
+		// Preferred translators(GET)
+		System.out.println("Account::PreferredTranslators(GET)");
+		try	{
+			response = this.gc.getAccountPreferredTranslators();
+			System.out.print("[OK] ");
+			System.out.println(verbose ? response.toString() : "");
+		} catch (GengoException x) {
+			System.out.print("[NG] ");
+			System.out.println(x.getMessage());
+		}
+		System.out.println("[testAccount] end@" + this.sessionHash);
+	}
+	
+	/**
+	 * Test service functionalities.
+	 */
+	public void testService(Boolean verbose) {
+		verbose = verbose == null ? true : verbose;
+		System.out.println("[testService] start@" + this.sessionHash);
+		JSONObject response;
+		// Language pairs(GET)
+		System.out.println("Service::Language pairs(GET)");
+		try	{
+			response = this.gc.getServiceLanguagePairs();
+			System.out.print("[OK]");
+			System.out.println(verbose ? response.toString() : "");
+		} catch (GengoException x) {
+			System.out.print("[NG]");
+			System.out.println(x.getMessage());
+		}
+		// Languages(GET)
+		System.out.println("Service::Languages(GET)");
+		try	{
+			response = this.gc.getServiceLanguages();
+			System.out.print("[OK]");
+			System.out.println(verbose ? response.toString() : "");
+		} catch (GengoException x) {
+			System.out.print("[NG]");
+			System.out.println(x.getMessage());
+		}
+		// Quote(POST)
+		System.out.println("Service::Quote(POST)");
+		try	{
+			List<TranslationJob> qt_jobs = new ArrayList<TranslationJob>();
+			qt_jobs.add(new TranslationJob("Test job for quoting.(1) (" + this.sessionHash + ")", "Test.",
```

Comment:
In this case `sessionHash` is defined as String. So I don't understand where is the `better` point.
Could you please explain where is it? Thanks.